src/PhpBrew/VariantBuilder.php

```diff
@@ -551,6 +552,27 @@ public function __construct()
             }
         };
 
+        $this->variants['sodium'] = function (Build $build, $prefix = null) {
+            if ($build->compareVersion('7.2') < 0) {
+                echo "Sodium is builded since PHP version 7.2.0, please install sodium as extension.\n";
```

Comment:
Sorry for the last minute request but this needs to be changed. "Builded" is not a word in English.

How about "Sodium is available as a core extension since PHP 7.2.0. Please use 'phpbrew ext install sodium' to install it from PECL"?

Reply:
I meant to say 'bundled' instead of 'builded'. I've changed it to your suggestion.

Reply:
I thought about it this way but "bundled" in this context means that the underlying C library (libsodium) is packaged along the PHP sources like it used to be done with libzip (see https://github.com/php/php-src/pull/1828) which is not true about libsodium and is not relevant in this case.